recipes/elm-format/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -ex
+
+if [ "$(uname)" == "Darwin" ]; then
+  # On Mac, we can use pre-built binaries
```

Comment:
Note that CI passing doesn't mean that it'll work on all OSX platforms we support since CI is using a OSX 10.15. (Linux on the other hand uses CentOS 6 which is the oldest we support).

Reply:
yeah, but we don't have haskell for mac, so I am not sure I could get it to compile from source :(

Reply:
Is that the `stack` package? It's available on OSX.

Reply:
Yeah, it is.  Unfortunately new versions of  stack (which this requires) use the newer GLIBC, so this will all have to wait for the new virtual package.

Reply:
Virtual package should be added as a `run_constrained` which is not used for conda versions that doesn't support it. IMO, it's fine to add a `run_constrained` and not wait for the new conda version.

Reply:
Sure, that makes sense. But it still needs the newer version of glibc around to pass the tests, since the stack binary itself links againts a new version of glibc